kmcd-prelude.md

```diff
@@ -16,29 +16,18 @@ module KMCD-PRELUDE
  // ---------------------------------------------------------------------
     rule DEPLOY-PRELUDE
       =>
-         // Contract Authorizations
-         // -----------------------
-
-         // Auhthorize Pot/End/Spot for Vat
-         transact ADMIN Vat . rely Pot
-         transact ADMIN Vat . rely End
-         transact ADMIN Vat . rely Spot
-
-         // Authorize End for Pot
-         transact ADMIN Pot . rely End
-
-         // Auhthorize Pot/End for Vow
-         transact ADMIN Vow . rely Pot
-         transact ADMIN Vow . rely End
+         // Main Deployments
+         // ----------------
 
-         // Authorize End for Cat/Pot
-         transact ADMIN Cat . rely End
+         transact ADMIN Gem "MKR" . init
+         transact ADMIN Deploy . deploy Gem "MKR"
 
-         // Authorize Vow for Flap
-         transact ADMIN Flap . rely Vow
+         transact ADMIN GemJoin "MKR" . constructor Vat Gem "MKR"
```

Comment:
We shouldn't need a `GemJoin` for MKR...it is not used as collateral (and never should be due to its special role in the protocol).

Reply:
Heh, seems like I missed this in prior reviews :)

Reply:
Indeed we don't need this, it's removed.